### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by vishal332008** [bombsquad-community/plugin-manager#40] [General]
Hello

**Comment by Loup-Garou911XD** [comment]
Hi

**Comment by Loup-Garou911XD** [comment]
Pr #65 pending for review, just saying, no hurry

**Comment by rikkolovescats** [comment]
hey guys, yep i'll get to it hopefully soon

**Comment by Loup-Garou911XD** [comment]
@Freaku17 hello there

**Comment by Loup-Garou911XD** [comment]
@Freaku17 is that you?
https://media.discordapp.net/attachments/588424130647294128/1110232140211310592/Screenshot_2023-05-22-21-08-56-83_572064f74bd5f9fa804b05334aa4f912.jpg

**Reply by Freaku17** [reply]
Lmao, ofcourse NOT.

Didn't know I'm so popular that people have started impersonating me 😂😂

Also a short notie here, IF I ever come back to Discord I'll link my GitHub account to it to verify my identity 👍

**Reply by Freaku17** [reply]
@Loup-Garou911XD

**Reply by Loup-Garou911XD** [reply]
> @Loup-Garou911XD

Yes okay,also I added your minigames to plugman

**Reply by Loup-Garou911XD** [reply]
Not the right place to ask but how was your 12th?

**Comment by HeyErfan** [comment]
**Hi**

**Reply by Loup-Garou911XD** [reply]
Hello

**Comment by AMB4324** [comment]
How we use 
tag

**Reply by Loup-Garou911XD** [reply]
What tag?

**Comment by AlysiaDavis** [comment]
New Plugin Spare Or Kill https://www.mediafire.com/file/7fop1larqogyr64/[spare_or_kill](https://www.mediafire.com/file/7fop1larqogyr64/spare_or_kill.py/file).py/file

**Reply by Loup-Garou911XD** [reply]
Hey👋 
Read [this](https://github.com/bombsquad-community/plugin-manager?tab=readme-ov-file#contributing) about how to submit a plugin
